upgrade-guide/en/index.md

```diff
@@ -0,0 +1,371 @@
+{:.tip}
+
+Backup the private file directory.
+
+```bash
+$ tar cvzf "$OJS_BACKUP_PATH/private-date.tgz" "$OJS_PRIVATE_PATH"
+```
+
+Backup the public files directory.
+
+```bash
+$ tar cvzf "$OJS_BACKUP_PATH/ojsfiles-date.tgz" "$OJS_WEB_PATH"
+```
+
+Backup any other customizations you have  made to the software, such as custom plugins or locale files.
+
+### 4. Create Sandbox
+
+Use your backup to create a sandbox environment and test the upgrade in that sandbox first. The steps below can be used in your sandbox environment to perform an upgrade.
+
+Once the test is complete, you can run any automated or manual tests you have configured to ensure the upgrade did not introduce regressions.
+
+Only perform the next steps on your live, production environment if you have already completed a test upgrade in your sandbox environment.
+
+### 5. Download Release Package
+
+Download the correct release package.
+
+```bash
+$ cd "$OJS_ROOT_PATH"
+$ wget "https://pkp.sfu.ca/ojs/download/$OJS_VERSION.tar.gz"
+```
+
+### 6. Check System Requirements
+
+Check the [README](https://pkp.sfu.ca/ojs/README) file from the downloaded `tar.gz` and be sure your system meets the following requirements.
+
+- Apache version
+- MySQL or PostgreSQL version
+- PHP version
+- Server libraries and module requirements
+
+In addition, you will want to perform the following checks.
+
+- Adjust your PHP timeouts and memory limits to ensure the upgrade process can complete successfully.
+- Check the server libraries and module requirements for any plugins you've added (these can often be found in the plugin's README file).
+
+### 7. Install Release Package
+
+Backup the application files.
+
+```bash
+$ mv "$OJS_WEB_PATH" "$OJS_BACKUP_PATH"
+```
+
+Extract the release package.
+
+```bash
+$ mkdir "$OJS_WEB_PATH"
+$ tar -xvz --strip-components=1 "$OJS_VERSION.tar.gz" -C "$OJS_WEB_PATH"
+```
+
+Restore the `config.inc.php` file.
+
+```bash
+$ cp "$OJS_BACKUP_PATH/html/config.inc.php" "$OJS_WEB_PATH"
+```
+
+Restore the `.htaccess` file if it exists.
+
+```bash
+$ cp "$OJS_BACKUP_PATH/.htaccess" "$OJS_WEB_PATH"
+```
+
+Run the following command to compare your configuration file with the template of the new release. Add or remove any configuration options as necessary.
+
+```bash
+$ diff "$OJS_BACKUP_PATH/config.inc.php" "$OJS_WEB_PATH/config.TEMPLATE.inc.php"
+```
+
+> You can review `docs/release-notes` to learn about important changes introduced in each version. The `config.TEMPLATE.inc.php` includes a description for most configuration parameters.
+{:.tip}
+
+Restore the public files.
+
+```bash
+$ cp -r "$OJS_BACKUP_PATH/html/public/*" "$OJS_PUBLIC_PATH"
+```
+
+Set the permissions of new files as required by your server configuration.
+
+```bash
+(Debian)$ sudo chown -R nobody:www-data /var/www/html/public/ /var/www/html/cache/
```

Comment:
Debian part need to be:
 (Debian)$ sudo chown -R WEB_USER:WEB_GROUP /var/www/html/public/ /var/www/html/cache/
 
And same on line  241.